pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -605,7 +637,44 @@ func DNSMonitor(bn string, bnNum int, ctx *zedrouterContext, status *types.Netwo
 	}
 	if status.Type == types.NetworkInstanceTypeSwitch {
 		switched = true
-		filter = "udp and (port 53 or port 67)"
+		filter = "icmp6 or (udp and (port 53 or port 67 or port 546 or port 547))"
```

Comment:
I suspect you want to check the icmp6 code to avoid having to parse e.g., icmp6 echo packets.

Reply:
I don't see a check for the icmp code field being Neighbor Solicitation